src/components/Header.js

```diff
@@ -5,14 +5,21 @@ import '../styles/Header.scss';
 export const Header = () => {
     return (
       <section class="header">
-        <h1>Ian Holladay</h1>
-        <h2>Web Developer</h2>
-            <Link to= '/projects' >
+        <section class="intros">
+          <h1 class="ian">Ian Holladay</h1>
+          <h2>Web Developer</h2>
```

Comment:
Do these need to be h1s? I feel like when I was messing with it, it didn't really affect anything.